apisix/cli/ops.lua

```diff
@@ -452,15 +449,13 @@ Please modify "admin_key" in conf/config.yaml .
                     port = 9080
                 end
 
-                if enable_http2 == nil then
-                    enable_http2 = false
-                end
-                if enable_http2 == true then
-                    enable_http2_global = true
+                if enable_http2 ~= nil then
+                    util.die("ERROR: enable_http2 is deprecated,"
+                            .. " you should use apisix.enable_http2.", "\n")
```

Comment:
We need a more explicit error log. For instance, the port level "enable_http2" in node_listen has been deprecated from the 3.9 version, and you should use "enable_http2" in the apisix level.